src/turing_inference.jl

```diff
@@ -39,13 +39,12 @@ function turing_inference(
                 push!(u0, convert(T,prob.u0[i]))
             end
         end
-        _saveat = isnothing(t) ? Float64[] : t
+        _saveat = t === nothing ? Float64[] : t
         sol = concrete_solve(prob, alg, u0, p; saveat = _saveat, progress = progress, save_idxs = save_idxs, kwargs...)
         failure = size(sol, 2) < length(_saveat)
 
         if failure
-            S = typeof(Turing.Inference.getlogp(_varinfo))
-            Turing.Inference.acclogp!(_varinfo, S(-Inf))
+            Turing.acclogp!(_varinfo, -Inf)
```

Comment:
Turing (or rather DynamicPPL) will call `S(-Inf)` internally to convert it to the type of `getlogp(_varinfo)`, so IMO DiffEqBayes should not have to care about it.

Reply:
Alright